_data/tags.yml

```diff
@@ -0,0 +1,69 @@
+# id: used in object metadata (tags-scopes, tags-topics)
+# name-short: shown in object cards
+# name-long: used in headings
+# index-weight: show topic on front page is greater than zero
+# description: description of the topic
+
+- id: cz
+  name-short: CZ
+  name-long: the Czech Republic
```

Comment:
I am not a language professional but I'd rather omit the "the" from name-long of the scope tags. To me, it looks silly on the topics page. Was this approved by or even pushed for by Jirka?